src/docs/concepts-api-sdk/rate-limiting/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Rate limiting in Marketplace API and Integration API
+slug: rate-limiting
+updated: 2022-12-21
+category: concepts-api-sdk
+ingress:
+  This article describes rate limiting in Marketplace API and
+  Integration API, as well as suggests how to handle those limits
+  gracefully.
+published: true
+---
+
+Starting in January 2023, Marketplace API and Integration API will
+feature new rate limits in dev and demo environments. In addition,
+Integration API will feature new concurrency limits in dev and demo
+environments and, eventually, in production environments.
+
+<extrainfo title="What are rate limits and concurrency limits?">
+
+**Rate limit** means a limit on the number of requests that the API can
+process within a time span, e.g. in one minute.
+
+**Concurrency limit** means a limit on the number of simultaneous
+requests that the API can process at any given moment.
+
+</extrainfo>
+
+The rate limits are different for queries (fetching data) and commands
+(modifying data). Queries are rate limited at 1 request per second (60
+requests per minute) on average. Commands are rate limited at 1 request
+per 2 seconds (30 requests per minute) on average. The rate limit
+applies per client IP address. You can find more information on Flex
+rate limits in the API reference for
+[Marketplace API](https://www.sharetribe.com/api-reference/marketplace.html#rate-limits)
+and
+[Integration API](https://www.sharetribe.com/api-reference/integration.html#rate-limits).
+
+It is good to note that **production environments are currently not rate
+limited**. Still, as you build your Flex marketplace implementation to
+take the rate and concurrency limits into account in your development
+environment, we do appreciate it if you also transfer those behaviors
+into production.
+
+## Rate limit interaction
```

Comment:
Comment: I find this title a bit confusing. I had to reread it couple of times before getting my head around what this paragraph is all about.

I guess the confusion came from me not realizing, interaction with whom?

My improvement suggestion would be e.g. "Interaction between different rate limits"

Reply:
Good point, fixed!